src/iop/iop_api.h

```diff
@@ -180,29 +180,29 @@ REQUIRED(void, process, struct dt_iop_module_t *self, struct dt_dev_pixelpipe_io
                         const struct dt_iop_roi_t *const roi_out);
 /** a tiling variant of process(). */
 DEFAULT(void, process_tiling, struct dt_iop_module_t *self, struct dt_dev_pixelpipe_iop_t *piece, const void *const i,
-                               void *const o, const struct dt_iop_roi_t *const roi_in,
-                               const struct dt_iop_roi_t *const roi_out, const int bpp);
+                              void *const o, const struct dt_iop_roi_t *const roi_in,
+                              const struct dt_iop_roi_t *const roi_out, const int bpp);
 
 #ifdef HAVE_OPENCL
 /** the opencl equivalent of process(). */
 OPTIONAL(int, process_cl, struct dt_iop_module_t *self, struct dt_dev_pixelpipe_iop_t *piece, cl_mem dev_in,
                           cl_mem dev_out, const struct dt_iop_roi_t *const roi_in,
                           const struct dt_iop_roi_t *const roi_out);
 /** a tiling variant of process_cl(). */
-OPTIONAL(int, process_tiling_cl, struct dt_iop_module_t *self, struct dt_dev_pixelpipe_iop_t *piece, const void *const i,
-                                 void *const o, const struct dt_iop_roi_t *const roi_in,
-                                 const struct dt_iop_roi_t *const roi_out, const int bpp);
+DEFAULT(int, process_tiling_cl, struct dt_iop_module_t *self, struct dt_dev_pixelpipe_iop_t *piece, const void *const i,
```

Comment:
Is this enough to set `default_process_tiling_cl()` ?

Reply:
Yep. Try renaming the declaration of `default_process_tiling_cl` in `tiling.h` to something else; the compiler won't like it.